Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/WANDPowderReduction.py

```diff
@@ -196,17 +217,20 @@ def PyExec(self):
                 EnableLogging=False,
             )
 
-            # calibration
-            if cal is not None:
-                _ws_cal_resampled = self._resample_calibration(_wsn, _mskn, xMin, xMax)
+            # TODO Add in if statement for `FilteredInputs`
+            if not filtered_eve:
```

Comment:
Isn't this TODO exactly what this PR does?

Same thing about flipping the if/else branches